rusk/Cargo.toml

```diff
@@ -72,8 +72,6 @@ node = { version = "0.1", path = "../node", optional = true }
 dusk-consensus = { version = "0.1.1-rc.3", path = "../consensus", optional = true }
 node-data = { version = "0.1", path = "../node-data", optional = true }
 
-## Bump to 0.8.7 requires rust 1.71.0 due to `build_hasher_simple_hash_one` feature stabilization
-ahash = "=0.8.6"
```

Comment:
@herr-seppia I removed this to fix an incompatibility with `metrics` crate and the CI passes. However, could you confirm it is truly redundant.